keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -0,0 +1,278 @@
+
+### Rollout, Upgrade and Rollback Planning
+
+*This section must be completed when targeting beta graduation to a release.*
+
+* **How can a rollout or rollback fail? Can it impact already running workloads?**
+  A feature gate `SeparateTaintManager` controls whether to use the split `TaintManager` or the old combined `NodeLifecycleController`.
+
+* **What specific metrics should inform a rollback?**
+  N/A
+
+* **Were upgrade and rollback tested? Was the upgrade→downgrade→upgrade path tested?**
+  They will be covered in unit and e2e tests.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?**
+  No
+
+### Monitoring Requirements
+
+*This section must be completed when targeting beta graduation to a release.*
+
+* **How can an operator determine if the feature is in use by workloads?**
+    * It can be determined by if the feature gate `SeparateTaintManager` is used.
+* **How can someone using this feature know that it is working for their instance?**
+    * Node taints and taint-based pod eviction work as usual. Admins and users should not see any different behavior if everything works as planned.
+* **What are the reasonable SLOs (Service Level Objectives) for the enhancement?**
+    * The performance of node taint-based eviction should remain the same level as before.
+* **What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?**
+    * The metrics for both `NodeLifecycleController` and `TaintManager`'s queues should stay the same levels as before.
+* **Are there any missing metrics that would be useful to have to improve observability of this feature?**
+    * No
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?** 
+    * N/A
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+    * No
+* **Will enabling / using this feature result in introducing new API types?** 
+    * No
+* **Will enabling / using this feature result in any new calls to the cloud provider?**
+    * No
+* **Will enabling / using this feature result in increasing size or count of the existing API objects?**
+    * No
+* **Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?**
+    * No
+* **Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?**
+    * No
+
+### Troubleshooting
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+    * The same behavior as the current version: fail to add taint and/or evict pods on tainted nodes 
+* **What are other known failure modes?**
+    * No
+* **What steps should be taken if SLOs are not being met to determine the problem?**
+    * If the default or custom taint-manager is working or not. 
```

Comment:
I don't understand it - can you please clarify?

Reply:
Updated to 

If the pod eviction latency increases significantly, validate if the communication between `NodeLifecycleController` and `TaintManager` works. If the number of pod evictions is abnormal, run tests to verify the TaintManager works properly.